assets/src/edit-story/components/library/panes/media/local/dropDownMenu.js

```diff
@@ -69,15 +77,19 @@ function DropDownMenu({
   onMenuSelected,
 }) {
   const options = [
-    { name: __('Edit', 'web-stories'), value: 'edit' },
-    { name: __('Delete', 'web-stories'), value: 'delete' },
+    {
+      group: [
+        { label: __('Edit', 'web-stories'), value: 'edit' },
```

Comment:
if you did want to make this a context menu, you'd just need to add `onClick` to each of these items and remove the `group`.

Then 

`<ContextMenu items={options} isOpen={isMenuOpen} />` 

Additionally, if you don't want to make this a context menu - if you remove the `group` here you'll get a simplified dropDown.

Reply:
Regarding your first point, I just tried that. But the context menu is displayed inline, which doesn't work as it has to show over several elements - and it also seems to be completely inaccessible by both mouse and keyboard in this usage?

There are actually two menus opened here - and the latter is all the way to the right in the media list only showing the first few pixels because it's not overlaying correctly.

![Screen Shot 2021-03-10 at 16 04 46](https://user-images.githubusercontent.com/637548/110697655-a78f0400-81ba-11eb-8f31-35234cddc391.png)

Regarding your second point, I tried that first but it crashed without the group wrapper. This doesn't seem to work:

```js
  const options = [
    { label: __('Edit', 'web-stories'), value: 'edit' },
    { label: __('Delete', 'web-stories'), value: 'delete' },
  ];
```

Crashes with:

```
Uncaught TypeError: Cannot read property 'map' of undefined
    at listGroupings.js:51
```

So I reverted to what I already had and it seems to work. If we want to change it in the future, we can do so in the API, so it would cover this usecase.

Reply:
weird! totally agree it's not worth the time right now. what you have works.